cmd/allocator/main.go

```diff
@@ -255,24 +282,10 @@ func readTLSCert() (*tls.Certificate, error) {
 // getServerOptions returns a list of GRPC server options.
 // Current options are TLS certs and opencensus stats handler.
 func (h *serviceHandler) getServerOptions() []grpc.ServerOption {
-	if h.tlsDisabled {
```

Comment:
Not sure I'm following why this was removed? I figure it's because the TLS details are above?

@pooneh-m will have better insight into if there is a better way to refactor this I expect.

Reply:
I have changed the code a bit after your comment as originally I incorrectly always called `ListenAndServeTLS` and now I call `ListenAndServe` when `h.tlsDisabled`.

Here I have changed the behaviour of `getServeroptions` in the past if `h.tlsDisabled` then KeepAlive and StatsHandler would not be added to the ServerOptions. I think it is ok to always have them whether TLS exists or not.
At this point `getServeroptions` is not aware of TLS as we manage it inside the `http.Server` `TLSConfig`